libraries/tracking/subscriptions/setup.js

```diff
@@ -26,6 +26,7 @@ export default function setup(subscribe) {
       type: get(clientInformationResponse, 'value.device.type', TYPE_PHONE),
       os: get(clientInformationResponse, 'value.device.os.platform', OS_ALL),
       state: getState(),
+      services: [get(clientInformationResponse, 'value.device.supportedAnalyticsServices', [])],
```

Comment:
you have now an array in an array here.
services: [[]].
please remove the outer array

Reply:
done